File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-persons-training-event_item.ts


import { IGetPersonsTrainingEvent_Item } from './iget-persons-training-event_item';

export class GetPersonsTrainingEvent_Item implements IGetPersonsTrainingEvent_Item {
  
	public TrainingEventID: number = 0;
	public Name: string = "";
	public PersonID: number = 0;
	public ParticipantType: string = "";
	public EventStartDate?: Date;
	public EventEndDate?: Date;
	public BusinessUnitAcronym: string = "";
	public TrainingEventRosterDistinction: string = "";
	public Certificate?: boolean;
	public TrainingEventStatus: string = "";
  
}

File: GTTS/INL.TrainingService.Models/GetPersonsTrainingEvent_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class GetPersonsTrainingEvent_Item : IGetPersonsTrainingEvent_Item
    {
        public long TrainingEventID { get; set; }
        public string Name { get; set; }
        public long PersonID { get; set; }
        public string ParticipantType { get; set; }
        public DateTime? EventStartDate { get; set; }
        public DateTime? EventEndDate { get; set; }
        public string BusinessUnitAcronym { get; set; }
        public string TrainingEventRosterDistinction { get; set; }
        public bool? Certificate { get; set; }
        public string TrainingEventStatus { get; set; }
    }
}
